### PR TITLE
[dev-v2.5] Add  v1.20.11-rancher1-1, v1.19.15-rancher1-1

### PR DESCRIPTION
**Pull request by doflamingo721**
https://github.com/rancher/rancher/issues/34632

### CODE REVIEW

**Review by kinarashah — DISMISSED** [review]

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -5344,6 +5344,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.5.10
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go]: Due to the OOB release and not waiting for v2.5.10, this comment will need to be `Enabled out of band post v2.5.9`

File: rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -5604,6 +5647,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
...
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go]: Due to the OOB release and not waiting for v2.5.10, this comment will need to be `Enabled out of band post v2.5.9`

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: rke/k8s_rke_system_images.go (hunk 1 above)
Comment (doflamingo721) [reply rke/k8s_rke_system_images.go]: Updated. Please check.

File: rke/k8s_rke_system_images.go (hunk 2 above)
Comment (doflamingo721) [reply rke/k8s_rke_system_images.go]: Updated. Please check.

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**